### DISCUSSION TITLE
Chrome store description translation

### DISCUSSION THREAD

**Discussion by bwisely** [k-ivan/visual-bookmarks-chrome#178] [General]
The visual bookmarks page replaces the default new tab.
視覺書籤可以取代原本瀏覽器預設的頁面。

Bookmarks page, includes searching, adding, editing, deleting and dragging bookmarks. The ability to generate thumbnails and use your own background image. Great possibility of customization and using your own styles.
視覺書籤頁面，包括可以搜尋、新增、編輯、刪除和拖曳書籤，並且能夠產生縮圖、或讓你自行選擇自己的背景圖。讓你更容易制定成你自己想要的風格。

Ideas, questions and suggestions https://github.com/k-ivan/visual-bookmarks-chrome/discussions
有任何想法、問題、以及建議請在 Github 發表 https://github.com/k-ivan/visual-bookmarks-chrome/discussions
Bugs and future requests https://github.com/k-ivan/visual-bookmarks-chrome/issues
有發現錯誤、或者功能上的建議，也請在 Github 上面提出疑問 https://github.com/k-ivan/visual-bookmarks-chrome/issues

**Comment by k-ivan** [comment]
Thanks for your help, will update in the next few days.

**Comment by k-ivan** [comment]
Hi @bwisely !

Updated description in web stores. Thank you!

**Comment by bwisely** [comment]
Add more and update some describes 
===========================
The visual bookmarks page replaces the default new tab.
視覺書籤可以取代原本瀏覽器預設的頁面。

Bookmarks page, includes searching, adding, editing, deleting and dragging bookmarks. The ability to generate thumbnails and use your own background image. Great possibility of customization and using your own styles.
視覺書籤，包括搜尋、新增、編輯、刪除和拖曳書籤等功能，並且能夠產生縮圖、或讓你自行選擇自己的背景圖。讓你更容易制定成你自己想要的風格。

Your URL will not be recorded. Your URL will follow your bookmarks. Visual bookmarks directly use one of the directories in your bookmarks as the basis. You can change which directory that you want to use as your main structure of the visual bookmark anytime.
且不會記錄你的網址，你的網址都是跟著你個人的書籤，視覺書籤是直接使用你瀏覽器書籤內的其中一個目錄當作基礎，你可以隨時更改要使用哪個目錄當作你的視覺書籤主架構。
The synchronization of visual bookmarks will only synchronize settings, and the data will only be recorded in your account.
視覺書籤的同步，只會同步設定，且資料只記錄在你自己的帳號內。


Ideas, questions and suggestions https://github.com/k-ivan/visual-bookmarks-chrome/discussions
有任何想法、問題、以及建議請在 Github 發表 https://github.com/k-ivan/visual-bookmarks-chrome/discussions
Bugs and future requests https://github.com/k-ivan/visual-bookmarks-chrome/issues
有發現錯誤、或者功能上的建議，也請在 Github 上面提出疑問 https://github.com/k-ivan/visual-bookmarks-chrome/issues

**Reply by k-ivan** [reply]
Give me some time, I really want to completely revise the entire description text. I want to provide a more detailed explanation of all the features and nuances for potential users. I might ask for help, and we will most likely replace the current text with a new one.
